node/service/src/chain_spec.rs

```diff
@@ -530,9 +544,9 @@ pub fn darwinia_build_spec_genesis() -> DarwiniaGenesisConfig {
 		darwinia_vesting: Some(darwinia_runtime::VestingConfig {
 			vesting: vec![
 				// Team vesting: 1 year aperiod start after 1 year since mainnet lanuch
-				(foundation_vesting, 365 * D_DAYS,  365 * D_DAYS, 0),
+				(team_vesting, 365 * D_DAYS, (5. * 365.25) as BlockNumber * D_DAYS, 0),
 				// Foundation vesting: 5 years period start when mainnet launch
-				(team_vesting, 0, (5. * 365.25) as BlockNumber * D_DAYS, 0)
+				(foundation_vesting, 0,  365 * D_DAYS, 0)
```

Comment:
Please re-check this. team vesting period is one year, foundation vesting period is 5 year.

It looks not right.